precompiles/ArbOwnerPublic.go

```diff
@@ -36,6 +36,16 @@ func (con ArbOwnerPublic) IsChainOwner(c ctx, evm mech, addr addr) (bool, error)
 	return c.State.ChainOwners().IsMember(addr)
 }
 
+// IsNativeTokenOwner checks if the account is a native token owner
+func (con ArbOwnerPublic) IsNativeTokenOwner(c ctx, evm mech, addr addr) (bool, error) {
```

Comment:
Ideally we should add some tests related to those new precompile methods.
We had an issue a while ago with improper types being used on precompiles signatures, that caused some issues in production, and since then we are trying to keep a high test coverage on precompiles.
We recently added a lot of precompile tests https://github.com/OffchainLabs/nitro/pull/2709

Reply:
Done.